backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/bottomsheet/BpkBottomSheet.kt

```diff
@@ -43,10 +46,22 @@ fun BpkBottomSheet(
     state: BpkBottomSheetState = rememberBpkBottomSheetState(),
     sheetGesturesEnabled: Boolean = true,
     peekHeight: Dp = DefaultSheetPeekHeight,
+    dragHandleStyle: BpkDragHandleStyle = BpkDragHandleStyle.Default,
     content: @Composable (PaddingValues) -> Unit,
 ) {
     BottomSheetScaffold(
-        sheetContent = sheetContent,
+        sheetContent = {
```

Comment:
can you extract this and reuse across both bottom sheets?

Reply:
On it!

Reply:
Done, will push it!